.github/workflows/transifex_push_pull.yml

```diff
@@ -0,0 +1,111 @@
+    ##########################################
+    - name: "Checkout"
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.branch }}
+
+    ##########################################
+    # Install dependencies                  #
+    ##########################################
+    - name: Install Dependencies
+      run: |
+        sudo apt-get install gettext libgettextpo-dev python3-dev python3-pip libffi-dev libxml2-dev libxslt-dev git
+        pip install lxml gitpython PyGithub termcolor
+
+    ##########################################
+    # Run composer                           #
+    ##########################################
+    - name: Install Composer Dependencies
+      uses: php-actions/composer@v6
+      with:
+        dev: yes
+
+
+    ##########################################
+    # Generate <plugin>.pot file             #
+    ##########################################
+    - name: "Generate <plugin>.pot file"
+      run: ./vendor/bin/extract-locales
+
+
+    ##########################################
+    # Push and Pull transifiex source        #
+    ##########################################
+    - name: Push source file (.pot)
+      uses: transifex/cli-action@v2
+      with:
+        args: push
+        token: ${{ secrets.tx_token }}
+
+    - name: Clean Transifex CLI
+      run: rm -fr /tmp/tx
+
+    - name: Pull source file (.po / .mo)
+      uses: transifex/cli-action@v2
+      with:
+        args: pull --force --all --minimum-perc=80
+        token: ${{ secrets.tx_token }}
+
+    ##########################################
+    # Generate .mo files                     #
+    ##########################################
+    - name: "Compil mo"
+      run: ./vendor/bin/plugin-release --compile-mo
+
+    ##########################################
+    # Generate random branch name            #
+    ##########################################
+    - name: "Generate target branch name"
+      run: echo BRANCH_NAME=transifex/i18n-updates-$(date +%s) >> $GITHUB_ENV
+
+    - name: "Check status"
+      id: "check-status"
+      run: |
+        if [[ `git status --porcelain locales/*.mo` ]]; then
+          echo "CREATE_PR=yes" >> $GITHUB_OUTPUT
+        else
+          echo "CREATE_PR=no" >> $GITHUB_OUTPUT
+        fi
+
+
+    ##########################################
+    # Create Pull Request                    #
+    ##########################################
+    - name: Create Pull Request
+      if: ${{ steps.check-status.outputs.CREATE_PR == 'yes' }}
+      id: cpr
+      uses: peter-evans/create-pull-request@v7
+      with:
+        commit-message: Update locales
+        committer: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+        author: ${{ github.actor }} <${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com>
+        signoff: false
+        branch: ${{ env.BRANCH_NAME }}
+        base: ${{ inputs.branch }}
```

Comment:
I think we could simplify it by using the current branch that triggered the workflow (that is the default value).

```suggestion
```

We could then remove the `with: ref: ${{ inputs.branch }}` param in the checkout step and also the `inputs:` instruction.